files/stat.py

```diff
@@ -69,6 +69,179 @@
+            sample: 1424348972.575
+        wusr:
+            description: Tells you if the owner has write permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: True
+        rusr:
+            description: Tells you if the owner has read permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: True
+        xusr:
+            description: Tells you if the owner has execute permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: True
+        wgrp:
+            description: Tells you if the owner's group has write permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: False
+        rgrp:
+            description: Tells you if the owner's group has read permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: True
+        xgrp:
+            description: Tells you if the owner's group has execute permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: True
+        woth:
+            description: Tells you if others have write permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: False
+        roth:
+            description: Tells you if others have read permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: True
+        xoth:
+            description: Tells you if others have execute permission
+            returned: success, path exists and user can read stats
+            type: boolean
+            sample: True
+        lnk_source:
+            description: Original path
+            returned: success, path exists and user can read stats and the path is a symbolic link
+            type: boolean
+            sample: True
+        md5:
+            description: md5 hash of the path
+            returned: success, path exists and user can read stats and path supports hashing and md5 is supported
+            type: boolean
+            sample: True
+        checksum:
+            description: hash of the path
+            returned: success, path exists and user can read stats and path supports hashing
+            type: boolean
+            sample: True
+        pw_name:
+            description: User name of owner
+            returned: success, path exists and user can read stats and installed python supports it
+            type: string
+            sample: httpd
+        gr_name:
+            description:
+            returned: success, path exists and user can read stats and installed python supports it
+            type: string
+            sample: www-data
+'''
```

Comment:
What about isuid and isgid? I had these documented:
isuid - invoking user's id matches path's user id
isgid - invoking group's id matches path's group id